hyp3lib/rtc2color.py

```diff
@@ -43,26 +68,30 @@ def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
   print('Estimated Total RAM usage = {} GB'.format(size*16))
 
   # Read full-pol image
-  print('Reading full-pol image (%s)' % fullpolFile)
+  print('Reading full-pol image (%s)' % out_tif)
   data = fullpol.GetRasterBand(1).ReadAsArray()
   cp = (data[:rows, :cols]).astype(np.float16)
   data = None
   cp[np.isnan(cp)] = 0
+  # FIXME: do we get negative powers from RTC?
   cp[cp < 0] = 0
   if cleanup == True:
     cp[cp < 0.0039811] = 0
+  # FIXME: Should this be applied *before* the above cleanup?
   if amp == True:
     cp = cp*cp
```

Comment:
Doing this after the cleanup means we're apply the threshold differently to power and amplitude! This should either be:
* applied before the cleanup
* the threshold should be changed depending on whether this is power or amplitude data

Reply:
Yes, we should convert amp->power before cleaning, and then use a `-48 db` threshold for cleanup.